src/ipc/ipc3/handler.c

```diff
@@ -34,7 +34,9 @@
 #include <sof/platform.h>
 #include <rtos/string.h>
 #include <rtos/string_macro.h>
+#if CONFIG_TRACE
```

Comment:
why don't put those ifdefs in dma_trace.h file? It would have a global effect, also if somebody use dma_trace.h in future

Reply:
ok, so its to help track impact of where its used whilst we transition code to native Zephyr. I'm good with this now, but we should end up with a global fix as @marcinszkudlinski suggested once transition work is complete.